build.gradle

```diff
@@ -87,7 +93,7 @@ plugins {
 
 group 'org.cru.android'
 version '5.0.19-SNAPSHOT'
-ext.versionCode = (grgit.log(includes:['HEAD']).size() * 2) + 4025200
+ext.versionCode = (grgit.log(includes:['HEAD']).size() * 2) + 4026000
```

Comment:
also, you reverted a versionCode change from develop, this should go back to the `4025200` number